stock_picking_batch_print_pickings/views/stock_picking_type_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+
+    <record
+        id="view_picking_type_form_inherit_stock_picking_batch_print_pickings"
+        model="ir.ui.view"
+    >
+        <field name="name">Operation Types Batch Print Pickings</field>
+        <field name="model">stock.picking.type</field>
+        <field
+            name="inherit_id"
+            ref="stock_picking_batch.view_picking_type_form_inherit"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='batch_auto_confirm']" position="after">
+                <label for="batch_print_pickings" string="Print Pickings" />
+                <div class="o_row">
+                    <field name="batch_print_pickings" />
+                    <label
+                        for="number_copies_pickings"
```

Comment:
**question(important):** Isn't this a bit over-engineered? 🤔 

When printing the PDF, I can always set the number of copies I want:
![imagen](https://github.com/OCA/stock-logistics-workflow/assets/973709/f1cbcf5f-1947-4042-b68e-feaf428cc7f7)

Is there any strong reason why do we have to duplicate this feature that most (if not all) printers already have?

Reply:
OK that makes sense of course.

Would you mind writing something similar in the `readme/CONTEXT.md` file please?